docs/source/user-guide/features.rst

```diff
@@ -0,0 +1,88 @@
+        up_sampling_method: none
+    attributions:
+        attribution_method: smoothgrad
+        attribution_strategy: true_label
+        sources:
+        - ilsvrc2012-small-sg/attribution/ilsvrc2012-small.smoothgrad.h5
+    analyses:
+        - analysis_method: spectral_analysis
+        sources:
+            - ilsvrc2012-small-sg/analysis/ilsvrc2012-small.smoothgrad.h5
+
+ViRelAy supports the loading of multiple project files at once. This can be helpful if multiple similar classifiers need to be compared. When running ViRelAy using its command line interface and the built-in HTTP server, multiple project files can be specified as arguments like so:
+
+.. code-block:: console
+
+    $ python -m virelay path/to/project-1.yaml path/to/project-2.yaml
+
+When running ViRelAy using Gunicorn, multiple project files can either be passed as arguments to the ``create_app`` function or the project files can be specified as a colon separated list in the ``VIRELAY_PROJECTS`` environment variable. The ``projects`` argument of the ``create_app`` function takes precedent over the environment variable.
+
+
+.. code-block:: console
+
+    $ # Specifying multiple projects using the projects argument of the create_app function
+    $ gunicorn -w 4 -b 127.0.0.1:8080 "virelay.application:create_app(projects=['path/to/project-1.yaml', 'path/to/project-2.yaml'])"
+
+    $ # Or using the VIRELAY_PROJECTS environment variable
+    $ export VIRELAY_PROJECTS="path/to/project-1.yaml:path/to/project-2.yaml"
+    $ gunicorn -w 4 -b 127.0.0.1:8080 "virelay.application:create_app()"
+
+In the ViRelAy user interface, all loaded projects are displayed at the very top next to the ViRelAy logo. Clicking the respective project name will navigate the user to the respective project's page. The project selection can be seen in Figure 1.
+
+.. figure:: ../../images/user-guide/features/virelay-ui-project-selection.png
+    :alt: Project Selection in the ViRelAy UI
+    :align: center
+
+    Figure 1: The project selection in the ViRelAy UI.
+
+For the currently selected project, the project name, the dataset name, and the model name are displayed in the footer of the ViRelAy UI in the project info pane, which can be seen in Figure 2.
+
+.. figure:: ../../images/user-guide/features/virelay-ui-project-info-pane.png
+    :alt: Project Info Pane in the ViRelAy UI
+    :align: center
+
+    Figure 2: The project info pane in the ViRelAy UI, which shows some basic information about the currently selected project.
+
+Analyses
+--------
+
+Clusterings & Embeddings
+------------------------
+
+Incl. categories; categories can be used for classes or concepts, for multi-class datasets, one category can also correspond to multiple classes.
+
+Navigating Embeddings
+---------------------
+
+Inputs, Attributions & Heatmaps
+-------------------------------
+
+Importing, Exporting & Sharing
+------------------------------
+
+
+TODO: Rename the UI elements according to the paper
```

Comment:
Is this planned in this PR or in a separate one?

Reply:
This is planned in this PR, I have worked on the document today, and the to do is now gone.